lib/plugin.js

```diff
@@ -226,28 +232,58 @@ internals.implementation = function (server, options) {
 };
 
 
+const getTokenFromCookie = (cookieHeader, cookieName) => {
```

Comment:
The hapi style guide does not allow defining new "globals". This should be assigned to `internals.getTokenFromCookie`.

Reply:
I was not sure if this was to be done. I thought this had something to do with using it elsewhere in the library. Although I admit it's my mistake that I did not read the styleguide, sorry

Reply:
Don't sweat it. We can fix it up if the patch is otherwise good.